src/components/autocomplete/Autocomplete.vue

```diff
@@ -21,7 +21,18 @@
             @keydown.native.enter.prevent="enterPressed"
             @keydown.native.up.prevent="keyArrows('up')"
             @keydown.native.down.prevent="keyArrows('down')"
-        />
+        >
+            <template v-if="showClearButton" slot="right-icon">
+                <b-icon
+                    class="is-right is-clickable"
+                    icon="times-circle"
```

Comment:
Add the icon to https://github.com/buefy/buefy/blob/39e918aa1fe574bcc8096436080d99a7029f8279/src/utils/icons.js#L24 in order to automatically swap from MDI to FA and vice versa